icons/area.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 16V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l7-4A2 2 0 0 0 21 16z" />
+  <path d="m3.3 7 7.7 4.4a1.82 1.82 0 0 0 2 0L20.7 7" />
```

Comment:
Version based on `layers-2` ![icon (58)](https://github.com/lucide-icons/lucide/assets/25524993/b6388353-59c0-49dd-8353-f8368c88d253)

Reply:
Also not sure if `area` is the best name.

Reply:
> Version based on `layers-2`

@jguddas Also nice! I’m fine with either, but maybe the cube version would be better for consistency… What do you think? @karsa-mistmere